src/main/cli/cli.c

```diff
@@ -1279,6 +1306,9 @@ static void cliSerialPassthrough(char *cmdline)
             ports[0].mode = cliParseSerialMode(tok);
             break;
         case 3:
+            ports[0].options = cliParseSerialOptions(tok);
```

Comment:
Adding new option will make passthrough command backward-incompatible. Also concatenating directly makes sense for mode (rx,tx,rxtx), but is not that great for options.
It would be probably best to merge options into mode string, probably using some separator character (`rxtx|bidir|inverted` ?)

Reply:
will change that

Reply:
done. its just important for the help text. functional would be eaven `rxtxbidir_ppinverted`
i tried: "serialpassthrough esc_sensor 2000000 rxtx|bidir_pp reset" worked well

Reply:
I think @ledvinap's idea here is to use `|` as a tokeniser, and parse the options token by token - this is way less messy than checking for string matches in a single string, which worked (poorly) for the `rx` / `tx` case, but will become a total mess if we keep adding options.